lib/tags.js

```diff
@@ -73,6 +73,8 @@ exports.parseTagLine = (line) => {
  * @public
  *
  * @param {String[]} footerTagLines - footer tag lines
+ * @param {Object} [options={}] - options
+ * @param {Boolean} [options.lowerCaseFooterTags=false] - lowercase footer tag keys
```

Comment:
Huh, how come the default is `false` here, when earlier it said the default was `true`?